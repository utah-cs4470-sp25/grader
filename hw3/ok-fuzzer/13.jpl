assert true, ""
let Y = true
show 576
time show [[[[1,3], 5.0], []]]
assert true, "b"
let t3B = true
