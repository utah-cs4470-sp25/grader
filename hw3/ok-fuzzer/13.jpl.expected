(AssertCmd (TrueExpr) "")
(LetCmd (VarLValue Y) (TrueExpr))
(ShowCmd (IntExpr 576))
(TimeCmd (ShowCmd (ArrayLiteralExpr (ArrayLiteralExpr (ArrayLiteralExpr (ArrayLiteralExpr (IntExpr 1) (IntExpr 3)) (FloatExpr 5)) (ArrayLiteralExpr)))))
(AssertCmd (TrueExpr) "b")
(LetCmd (VarLValue t3B) (TrueExpr))
